importscan/tests/test_importscan.py

```diff
@@ -174,3 +174,7 @@ def test_module_in_zipped():
     scan(moduleinzipped)
 
     assert fixtures.calls == 1
+
+
+def test_scan_from_name():
+    scan('email')
```

Comment:
This test doesn't seem to verify anything in particular. We should check whether it's been touched through import at least.

Reply:
Oh and I'd like to see a test for dotted names too, or is that not supported?